qlf/dashboard/bokeh/qasnr/main.py

```diff
@@ -87,11 +87,15 @@ def update(arm, spectrograph, expid):
 # configure bokeh widgets
 exposure = get_exposures()
 
+print("EXPOSURES: ", exposure)
```

Comment:
We should avoid debug `print` statements, if this info is important should go to the log.